### PR TITLE
Tweaks to pytorch experience

### PR DESCRIPTION
**Pull request by griverat**
This PR has some slight improvements to the pytorch experience. For the MNIST part of the workshop, it includes the output of the loss function and a sample plot of the model prediction. In the climate data part, there are tweaks to some titles and more details on the problem and dataset to use.

Furthermore, this PR also adds sections to the pytorch notebooks to make the navigation easier.

### CODE REVIEW

**Comment by philipmjohnson** [comment]
Merged!
